extensions/amp-story/1.0/amp-story-bling-link.css

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
At a high-level, I think we should decide exactly how templated these are supposed to be (perhaps in a separate PR).  Is it okay if a publisher slaps a 6px red border around it?  If they change the padding?  Add a box-shadow?

We might want to think about whether it's worth either:

- Putting this in a separate shadow DOM tree, or
- Think through some specific properties we'd like to enforce (even if we're enforcing their default state)